src/amoc_always_null_ssl_session_cache.erl

```diff
@@ -1,42 +0,0 @@
--module(amoc_always_null_ssl_session_cache).
```

Comment:
Why do we remove this file? This was a workaround for high memory consumption in scenarios doing lots of HTTPs requests. Was this improved in recent Erlang/OTP versions?

Reply:
this one is not used in docker container anyway. also it's better to move it into the `amoc-arsenal-xmpp`